docs/sematic-vs.md

```diff
@@ -43,7 +43,7 @@ the bounds prescribed by the framework is simply not possible.
 At Sematic, we strive to hit the right balance of constraints to make work fast
 and effortless, and flexibility to not hinder our users' creativity.
 
-## Sematic vs. _____
+## Sematic vs. ...
```

Comment:
I switched to this `...` format to make it more clear that the sections below are shorthand for `Sematic Vs <X>`